tests/vmi_multus_test.go

```diff
@@ -908,6 +908,31 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 	Expect(err).ToNot(HaveOccurred(), "Failed to set interface %s up on VMI %s", interfaceName, vmi.Name)
 }
 
+func configSRIOVInterface(vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress, prompt string) {
+	interfaceName := fmt.Sprintf("$(ip --brief l | grep %s | cut -f1 -d' ')", interfaceMac)
+	cmdCheck := fmt.Sprintf("ip addr add %s dev %s\n", interfaceAddress, interfaceName)
+	err := tests.CheckForTextExpecter(vmi, []expect.Batcher{
+		&expect.BSnd{S: "\n"},
+		&expect.BExp{R: prompt},
```

Comment:
Could you make use of `tests.PromptExpression` instead of `prompt` parameter?

Reply:
Done.